main.tf

```diff
@@ -34,8 +34,9 @@ module "deployment" {
   #
   wait_for_rollout = false
 
-  source  = "terraform-iaac/deployment/kubernetes"
-  version = "1.4.2"
+  # source  = "terraform-iaac/deployment/kubernetes"
+  # version = "1.4.2"
```

Comment:
Use readable comment to clarify the source

Reply:
Additional comments have been added @gitlawr

Reply:
Comments update again @gitlawr